src/components/layout/layout.tsx

```diff
@@ -8,6 +8,7 @@ import ExternalLink from '../external-link';
 import Button from '../button';
 import ErrorReportingNotice from './error-reporting-notice';
 import classNames from 'classnames';
+import { DesktopMenu } from '../menu';
```

Comment:
I needed to split the menu because the texture was overlapping it and z-index trick didn't work anymrore.